examples/siro_sandbox/README.md

```diff
@@ -0,0 +1,43 @@
+# Sandbox Tool
+
+![siro_sandbox_screenshot](https://user-images.githubusercontent.com/6557808/230213487-f4812c2f-ec7f-4d68-9bbe-0b65687f769b.png)
+
+## Overview
+This is a 3D interactive GUI app for testing various pieces of SIRo, e.g. rearrangement episode datasets, Fetch and Spot robots, humanoids (controllers, animation, skinning), trained agent policies, batch rendering and other visualization.
+
+## Known Issues
+* The policy-driven agent doesn't seem to be working in terms of producing interesting actions. As a placeholder, I've injected random-base-movement behavior in `BaselinesController.act`; see comment "temp do random base actions".
+* One-time visual flicker shortly after app startup on Mac
+
+## Running HITL eval with a user-controlled humanoid and policy-driven Fetch
+
+1. Make sure you've followed the [SIRo install instructions](../../SIRO_README.md#installation).
+2. Run this command:
+```
+HABITAT_SIM_LOG=quiet python examples/siro_sandbox/sandbox_app.py --disable-inverse-kinematics --humanoid-user-agent --cfg habitat-lab/habitat/config/benchmark/rearrange/rearrange_easy_human_and_fetch.yaml --never-end habitat.dataset.split=minival
```

Comment:
`habitat-lab/habitat/config/` config path prefix can be removed. Hydra searches config files relative to `habitat-lab/habitat/config` directory.